### DISCUSSION TITLE
JK BMS BLE - Sensors update time

### DISCUSSION THREAD

**Discussion by mariusghiata** [Sleeper85/esphome-jk-bms-can#47] [General]
Hello,
I have connected my DEYE SUN-8k-SG01LP1-EU to my JK BMS using ESP32_Li-ion_BLE_jk-bms-can.yaml. After a few hours I notice that my sensors were updating after a long time. 
For example discharging_power sensor last update 30min ago (even if the power consumption is different every 10 seconds). 

Before using this yaml for the CAN communication I used another the yaml from syssi (same with BLE but no CAN) and the sensors were updating every 2-5 second or even instantly. Even the cell voltage / resistance were updating instantly.

What do you recommend should I do (or what did I do wrong?).

Please if you can help me in beginner language :)

Thank you,
![Screenshot 2024-04-19 at 22 52 57](https://github.com/Sleeper85/esphome-jk-bms-can/assets/120492446/dcc3d30e-51cb-49d0-8001-e3341302cf21)
![Screenshot 2024-04-19 at 22 54 41](https://github.com/Sleeper85/esphome-jk-bms-can/assets/120492446/e4c13ce7-6406-4ae6-9e72-fab64e76ef7d)

**Comment by Sleeper85** [comment]
Hi,

Replace : 
```YAML
logger:
  level: INFO
```

By : 
```YAML
logger:
  level: DEBUG
```

Some people have had similar issues with INFO mode.

**Reply by mariusghiata** [reply]
It worked. Thank you!

**Comment by shvmm** [comment]
> Hello, I have connected my DEYE SUN-8k-SG01LP1-EU to my JK BMS using ESP32_Li-ion_BLE_jk-bms-can.yaml. After a few hours I notice that my sensors were updating after a long time. For example discharging_power sensor last update 30min ago (even if the power consumption is different every 10 seconds).
> 
> Before using this yaml for the CAN communication I used another the yaml from syssi (same with BLE but no CAN) and the sensors were updating every 2-5 second or even instantly. Even the cell voltage / resistance were updating instantly.
> 
> What do you recommend should I do (or what did I do wrong?).
> 
> Please if you can help me in beginner language :)
> 
> Thank you, ![Screenshot 2024-04-19 at 22 52 57](https://private-user-images.githubusercontent.com/120492446/324101578-dcc3d30e-51cb-49d0-8001-e3341302cf21.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM2MDA2NzQsIm5iZiI6MTcxMzYwMDM3NCwicGF0aCI6Ii8xMjA0OTI0NDYvMzI0MTAxNTc4LWRjYzNkMzBlLTUxY2ItNDlkMC04MDAxLWUzMzQxMzAyY2YyMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQyMFQwODA2MTRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00MWVjMDhjYzYxNjk3N2RhOTIzYTlkMGY5OTc2YTk5ZGIzMjMyYTA0ZjA3MTI4Zjk2OWM2ZjkzY2YxOTc5Y2IxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.srLAl00L7fEaeRLo9OfS4tE_M6XbqbpfdtMa5FjwVM8) ![Screenshot 2024-04-19 at 22 54 41](https://private-user-images.githubusercontent.com/120492446/324101581-e4c13ce7-6406-4ae6-9e72-fab64e76ef7d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM2MDA2NzQsIm5iZiI6MTcxMzYwMDM3NCwicGF0aCI6Ii8xMjA0OTI0NDYvMzI0MTAxNTgxLWU0YzEzY2U3LTY0MDYtNGFlNi05ZTcyLWZhYjY0ZTc2ZWY3ZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQyMFQwODA2MTRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kNjhhZGVlYmVkYzQyNDUwMWI1MTk4NmQ2Zjc4NTUwZTE1MTczYTlkMGM2YWI1MzFjNTUwY2RjOGM2MjE1MGNiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.tiu2omQqn9CImf8DF9Au3vHCToWeEr1HXKgS37J7jSU)

I'm also facing the same issue as you with HA sensors not updating as frequently as they used to.

In my production system, I was using LOGGER Level: ERROR
While LOGGING LEVELS being a likely culprit, it may be possible that other changes made previously might be behind this.
For example, recent release of Home assistant has wonky behaviour of sliders and date-pickers, even on new installations.

**Reply by shvmm** [reply]
Also, another question to ask: Did your HA install got updated recently??
